internal/parse/parser.go

```diff
@@ -231,6 +237,39 @@ func (p *Parser) skipName() bool {
 	return p.pos > start
 }
 
+// isEscaped checks if the byte at p.pos is escaped.
+func (p *Parser) isEscaped() bool {
+	return p.peekByteAt('\\', p.pos-1) && !p.peekByteAt('\\', p.pos-2)
+}
+
+// skipCharFind advances the parser until it finds the char passed as a
+// parameter then jumps over it and returns true. If the parameter is escaped
+// the it is ignored.  If the end of the string is reached and no matching byte
+// was found, it returns false.
+func (p *Parser) skipCharFind(c byte) bool {
+	mark := p.pos
+	for p.pos < len(p.input) {
+		if p.peekByte(c) && !p.isEscaped() {
+			p.pos++
+			return true
+		}
+		p.pos++
+	}
+	p.pos = mark
+	return false
+}
+
+// skipChar jumps over the current char if it matches the byte passed as a
+// parameter. If the char is escaped it is ignored. Returns true in that case,
+// false otherwise.
+func (p *Parser) skipChar(c byte) bool {
```

Comment:
Maybe we could call <peekByte()> here, like so:

`
if p.peekByte(c) && (!p.isEscaped()) { `

could also do this at the <skipByte()> method, like so:

```
func (p *Parser) skipByte(b byte) bool {
	if p.peekByte(b) { 
```

Reply:
Nice, I've changed skipChar. I'll leave skipByte for now since its not in this PRs remit but its a good point.